modal/functions.py

```diff
@@ -886,6 +886,8 @@ async def _load(self: _Function, resolver: Resolver, existing_object_id: Optiona
                         is_method=function_definition.is_method,
                         use_function_id=function_definition.use_function_id,
                         use_method_name=function_definition.use_method_name,
+                        method_definitions=method_definitions,
```

Comment:
[nit] Can we make this look like `method_definitions=function_definition.method_definitions`, ditto for the other field. If it helps, after the FunctionData migration, we'll be able to have just one form this in the client - i.e. filling out the FunctionData proto directly. 

In the interm - we have this code in the server. We could pull it into the client and use that instead:

https://github.com/modal-labs/modal/blob/159c468655850cd8f53d6506bc3138b9e29c4e58/server/modal_server/function_definition.py#L89.

Reply:
Dumb on my part for not noticing this.